doc_admin/AdminManual.adoc

```diff
@@ -348,16 +415,16 @@ image:image17.jpeg[Splash.jpg,width=316,height=237]
 
 And then the main Menu:
 
-image:image18.png[MainMenu.PNG,width=235,height=456]
+image:image18.png[MainMenu.png,width=235,height=456]
 
 It is possible to create a shortcut (link) to the executable script on the desktop, in the Programs Menu or wherever is useful by using the provided icon “oh.ico” that is found in the package.
 
 image:image100.png[oh.ico,width=32,height=32]
 
 
-=== 3.2 Open Hospital - Portable mode
+=== 2.2 OH - PORTABLE mode
 
-Open Hospital is also distributed in a *PORTABLE* mode.
+Open Hospital can also be executed in a *PORTABLE* mode. A "portable" version is also distributed, in order to have all the needed software in one single package.
```

Comment:
These two sentences need work.  First it says you can "execute" in PORTABLE mode and then the second sentence says a "portable" version is *also* distributed.   It almost sounds like there is a difference between upper case PORTABLE and lowercase portable. 😃

Reply:
Could you please re-review this ? Thank you !

Reply:
is the new version ok? can I merge?